tests/utils.go

```diff
@@ -2723,8 +2728,25 @@ func NewBool(x bool) *bool {
 	return &x
 }
 
-func RenderJob(name string, cmd []string, args []string) *k8sv1.Pod {
-	job := k8sv1.Pod{
+func RenderJob(name string, cmd, args []string) *batchv1.Job {
```

Comment:
Please do not add any more code to `utils`, just open a new file `job.go` and place all job related stuff there (you can just add the new stuff, it is not a must to move everything now).